src/Viewer.js

```diff
@@ -1083,6 +1250,31 @@ export class Viewer {
         throw new Error(`Viewer::downloadSplatSceneToSplatBuffer -> File format not supported: ${path}`);
     }
 
+    /**
+     * Load a splat scene from an ArrayBuffer and convert to SplatBuffer instance.
+     * @param {ArrayBuffer} fileData The data of the file
+     * @param {number} splatAlphaRemovalThreshold Ignore any splats with an alpha less than the specified
+     *                                            value (valid range: 0 - 255), defaults to 1
+     *
+     * @param {function} onProgress Function to be called as file data are received
+     * @param {boolean} progressiveBuild Construct file sections into splat buffers as they are downloaded
+     * @param {function} onSectionBuilt Function to be called when new section is added to the file
+     * @param {string} format File format of the scene
+     * @return {AbortablePromise}
+     */
+    loadSplatSceneFromFileToSplatBuffer(fileData, splatAlphaRemovalThreshold = 1, onProgress = undefined,
+        progressiveBuild = false, onSectionBuilt = undefined, format) {
+        if (format === SceneFormat.Splat) {
+            return SplatLoader.loadFromFileData(fileData, splatAlphaRemovalThreshold, 0, false);
```

Comment:
why not passing the compression level? rather than magic `0`?

Also, I suggest this function to handle `inMemoryCompressionLevel` as an option parameter with some default. 

See that the viewer otherwise (URL loading feature) supports it as an option.